lib/verdict/experiment.rb

```diff
@@ -2,7 +2,7 @@ class Verdict::Experiment
 
   include Verdict::Metadata
 
-  attr_reader :handle, :qualifiers, :storage, :event_logger
+  attr_reader :handle, :qualifiers, :event_logger
```

Comment:
Should we mention this in the change log?

Reply:
Seems like a potential breaking change.

Reply:
I removed this because the storage reader is being overridden here: https://github.com/Shopify/verdict/blob/43a001e962add301237551df72f774373eaa8559/lib/verdict/experiment.rb#L73